src/GuardClauses.UnitTests/GuardAgainsOutOfRange.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Ardalis.GuardClauses;
+using System;
+using Xunit;
+
+namespace GuardClauses.UnitTests
+{
+    public class GuardAgainsOutOfRange
+    {
+        [Theory]
+        [InlineData(1, 1, 5)]
+        [InlineData(2, 1, 5)]
+        [InlineData(3, 1, 5)]
```

Comment:
When I said 1 to 3 I meant [InlineData(1, 1, 3)], [InlineData(2, 1, 3)], and [InlineData(3, 1, 3)]

This would then test all of the valid cases:
- Input matches lower bound
- Input is between bounds
- Input matches upper bound